macos/FluentUI/AvatarView/AvatarView.swift

```diff
@@ -325,11 +325,31 @@ open class AvatarView: NSView {
 	///
 	/// - note: Internal visibility exists only for unit testing
 	@objc(getColorForIndex:)
+	@available(*, deprecated, message: "Use getInitialsColorSetFromPrimaryText:secondaryText: instead")
 	public static func getColor(for index: Int) -> ColorSet {
 		let avatarBackgroundColors = AvatarView.avatarColors
 		return avatarBackgroundColors[index % avatarBackgroundColors.count]
 	}
 
+	/// Returns a color table index for a given display name
```

Comment:
Is this a table index or just a hash?
The Int returned by this function would have to divided by the table entry count and the remainder (mod) of it would be the index of the table, correct?

Reply:
yes, I didn't alter the original comment but I think the author's intent was that.

Reply:
Let's make the comment accurate with what the code is doing!

Reply:
I have updated the description of the method, the mod usage is already described in note section of the comment.